lib/ex_doc/formatter/html/templates/css/style.css

```diff
@@ -404,9 +410,9 @@ li.r2 {
 }
 
 div.breadcrumbs {
-  padding: 3px 6px;
+  padding: 3px 15px;
   background-color: #EEE;
-  margin: 0 -15px 10px;
+  margin: 0 -28px 10px;
```

Comment:
Why do you want to add more space? this change generate the following problem:

![capture](https://cloud.githubusercontent.com/assets/34700/9145474/4e57576a-3d1a-11e5-9e18-b43eb71b585d.png)